Dockerfile

```diff
@@ -65,31 +61,38 @@ RUN apt-get update && apt-get install -y --no-install-recommends  \
     vim                   \
     wget                  \
     zip                   \
-  && rm -rf /var/lib/apt/lists/* \
-  && apt-get clean all
```

Comment:
I guess this should be brought back (to keep the image small)

Reply:
That's true. Added.